README.md

```diff
@@ -31,23 +31,227 @@ privileges and forfeiture of registration fee.
+
+4. Connect to your Decred wallet. The client will auto-fill most of your wallet settings, but you will need to specify the account name. If you haven't already, follow the instructions above to create a trading account. If you really, really want to trade on the default wallet account, the account name is `default`. Enter the wallet password, which is the password you set up with **dcrwallet**. Enter the app password you created in step 3.
+
+<img src="docs/images/decred-reg.png" width="250">
+
+5. Enter the dex address, probably **dex.decred.org**.
+
+<img src="docs/images/add-dex-reg.png" width="250">
+
+6. Check the registration fee, and enter your password one more time to authorize payment.
+
+<img src="docs/images/confirm-reg.png" width="250">
+
+7. On the **markets view**, while you're waiting for confirmations on your registration fee, add a Bitcoin wallet. You'll need to specify the wallet name. If you haven't already, follow the instructions above to create a trading wallet. If you really, really want to trade on the default wallet, leave the wallet name blank. Enter the wallet password you set up with **bitcoin-cli**. If you really, really want to trade on an unencrypted wallet, you can leave the wallet password blank. Enter the app password you created in step 3.
+
+<img src="docs/images/create-btc.png" width="250">
+
+8. And that's it! Once your registration fee has enough confirmations, you can begin trading.
+
+
+## Important Stuff to Know
+
+Trades settle on-chain and require block confirmations. Trades do not settle instantly.
+In some cases, they may take hours to settle.
+**The client software should not be shut down until you are absolutely certain that your trades have settled**.
+
+**The client has to stay connected for the full duration of trade settlement**.
+Losses of connectivity of a couple minutes are fine, but don't push it.
+A loss of internet connectivity for more than 20 hours during trade settlement has the potential to result in lost funds.
+Simply losing your connection to the DEX server does not put funds at risk.
+You would have to lose connection to an entire blockchain network.
+
+**There are initially limits on the amount of ordering you can do**. 
+We'll get these limits displayed somewhere soon, but in the meantime,
+start with some smaller orders to build up your reputation. As you complete
+orders, your limit will go up.
+
+**If your account is suspended**, you can appeal the suspension.
+You may be asked to provide client log files to the operator for review.
+For dex.decred.org, reach out 
+[on Element](https://matrix.to/#/!mlRZqBtfWHrcmgdTWB:decred.org?via=decred.org&via=matrix.org&via=planetdecred.org)
+to appeal.
+
+
+## Advanced Client Installation
+
+### Dependencies
+
+1. [Go >= 1.14](https://golang.org/doc/install)
+2. [Node 12+](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) is used to bundle resources for the browser interface.
```

Comment:
I'm of the opinion that we should never miss an opportunity to stress that Node is not a runtime dependency.